app/mesh/1.6.x/installation/ecs.md

```diff
@@ -0,0 +1,151 @@
+Our example repository [launches the control plane with two additional containers](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/controlplane.yaml#L358-L387)
+that handle fetching this global secret and
+[covers bootstrapping this controller and running it as an ECS task](https://github.com/Kong/kong-mesh-ecs/blob/main/README.md#ecs-controller).
+
+Any option that enables operators to query the API from `localhost` (e.g. an SSH
+container in the task) will allow extracting the admin token.
+
+Once we have `kumactl` configured with this token, we can generate a new user
+token for the controller with a command like:
+
+```
+kumactl generate user-token --name ecs-controller --group mesh-system:admin --valid-for 8766h
+```
+
+We can configure our controller with this token using:
+
+- the `KUMA_API_TOKEN` environment variable
+
+We recommend communicating with the Kong Mesh API over TLS (served on port `5682` by default) and the controller supports:
+
+- the `KUMA_API_CA_BYTES` environment variable for passing the CA used to verify the TLS certificates presented by the API
+
+#### IAM permissions
+
+The controller uses the AWS API and its ECS task role must be authorized to perform:
+
+- `ecs:ListTasks` and `ecs:DescribeTasks`
+- `secretsmanager:GetSecretValue` and `secretsmanager:PutSecretValue`
+
+These permissions can of course be further restricted by including a `Resource` or `Condition` in
+the IAM policy statements. To make this easier, the controller supports:
+
+- a `--secret-name-prefix` command line switch to prefix the names of the
+  AWS secrets under which it saves tokens
+
+See [the example controller Cloudformation](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/controller.yaml) for how all of this ties together.
+
+## Services
+
+When deploying an ECS task to be included in the mesh, the following must be
+considered.
+
+### Outbounds
+
+Like on Universal, services are bootstrapped with a `Dataplane` specification.
+
+Transparent proxy is not supported on ECS and so the `Dataplane` resources for a
+service must enumerate all other mesh services that service contacts and include them
+[in `Dataplane` specification as `outbounds`](https://kuma.io/docs/1.4.x/documentation/dps-and-data-model/#dataplane-specification).
+
+See the example repository for [how to handle the Dataplane template
+with Cloudformation](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/counter-demo/demo-app.yaml#L30-L46).
+
+### `kuma.io/service` tag
+
+Every ECS task must be tagged with the `kuma.io/service` tag in
+order for the controller to include the task in the mesh. The ECS task
+authenticates to the mesh as this service and it should match the
+`kuma.io/service` value in the `Dataplane` resource.
+
+### Sidecar
+
+The sidecar must run as a container in the ECS task and be passed the AWS secret
+that holds the dataplane token created by the ECS controller.
+
+The controller _does not create_ the secret, it only puts and gets it. The
```

Comment:
```suggestion
The controller _does not create_ the secret, it only PUTs and GETs it. The
```
If this is referring to requests, then capitalize the terms.

Reply:
hmm it's referring to AWS API actions

Reply:
So it's a GET or a PUT request from the AWS API? Or is it something else?

Reply:
It's the PutSecretValue and GetSecretValue actions which don't map to HTTP methods

Reply:
Got it. Feel free to ignore, then :)